Source/Core/Core/NetPlayServer.cpp

```diff
@@ -1551,72 +1584,117 @@ void NetPlayServer::AbortGameStart()
 }
 
 // called from ---GUI--- thread
-bool NetPlayServer::SyncSaveData()
+bool NetPlayServer::CollectSaveSyncInfo(SaveSyncInfo* sync_info)
 {
-  // We're about to sync saves, so set m_saves_synced to false (waits to start game)
-  m_saves_synced = false;
-
-  m_save_data_synced_players = 0;
-
   u8 save_count = 0;
```

Comment:
I realise this is existing code, but why is this an u8? Seems like int or size_t would be a more natural choice?

Reply:
I'm guessing it was like that because it's sent over the network, so it needs a fixed size, and it never exceeds 7 (2 GC slots, all Wii stuff just counts as 1, and 4 GBA slots). It's then used as a counter for how many 'saves' have successfully synced on the client side. An odd variable in general really, there's probably better ways to handle this, not sure how much I want to mess with this logic here though...